client/js/components/common/feedback_icon.jsx

```diff
@@ -0,0 +1,41 @@
+
+"use strict";
+
+import React                  from "react";
+
+import { CORRECT, INCORRECT, UNGRADED } from "../assessments/universal_input";
+
+export default class FeedbackIcon extends React.Component{
+
+  static propTypes = {
+    // Whether answer is correct, incorrect, or has not been graded
+    // Should be one of CORRECT, INCORRECT, UNGRADED.
+    gradeState: React.PropTypes.number.isRequired
+  };
+
+  render(){
+    var content = null;
```

Comment:
`content` is always "definitely assigned" so this initial assignment is just noise.